webroot/index.php

```diff
@@ -5,6 +5,13 @@
 require_once $LOC_HEADER;
 ?>
 
+<br><br>
```

Comment:
```suggestion

```
In general I don't like using line breaks to add spacing. In this case I'd remove these breaks. If for whatever reason there needs to be extra spaces on top, it can be changed in the content management page.